modelscan/settings.py

```diff
@@ -1,18 +1,23 @@
 import tomlkit
 
-from enum import Enum
 from typing import Any
 
 from modelscan._version import __version__
 
 
-class DefaultModelFormats(Enum):
-    TENSORFLOW = "tensorflow"
-    KERAS_H5 = "keras_h5"
-    KERAS = "keras"
-    NUMPY = "numpy"
-    PYTORCH = "pytorch"
-    PICKLE = "pickle"
+class Property:
+    def __init__(self, name: str, value: Any) -> None:
+        self.name = name
+        self.value = value
+
+
+class SupportedModelFormats:
+    TENSORFLOW = Property("TENSORFLOW", "tensorflow")
+    KERAS_H5 = Property("KERAS_H5", "keras_h5")
+    KERAS = Property("KERAS", "keras")
+    NUMPY = Property("NUMPY", "numpy")
+    PYTORCH = Property("PYTORCH", "pytorch")
+    PICKLE = Property("PICKLE", "pickle")
```

Comment:
Enum already supports name and value.

I'd recommend using adding support for str, if you want to enable additaion behavior like f string support.

<img width="428" alt="Screenshot 2024-05-23 at 4 53 26 PM" src="https://github.com/protectai/modelscan/assets/8380706/66b8af9c-9d6a-414f-a302-ffad3daddcab">

Reply:
Replacing the enum since it's not extendable, in the current enum state other applications adding their own scanners can't add additional model formats or issue codes